include/internal/ktls.h

```diff
@@ -197,7 +197,7 @@ static ossl_inline ossl_ssize_t ktls_sendfile(int s, int fd, off_t off,
 
     ret = sendfile(fd, s, off, size, NULL, &sbytes, flags);
     if (ret == -1) {
-	    if (errno == EAGAIN && sbytes != 0)
+	    if ((errno == EAGAIN || errno == EBUSY) && sbytes != 0)
```

Comment:
Looking at the manpage and the implementation in the kernel, other errors such as EINTR and ENOBUFS may also do partial sends.  I believe we should return `sbytes` anytime the value is not 0, that is:

```
    if (ret == -1 && sbytes == 0)
        return -1;
    return sbytes;
```

Reply:
The man page could perhaps be updated in a few places, e.g., the EBUSY case just says "partial data may have been sent" but nothing about `*sbytes`.